### PR TITLE
BUG 2013199: Update operator bundle

### PR DESCRIPTION
**Pull request by pliurh**

### CODE REVIEW

File: manifests/4.10/supported-nic-ids_v1_configmap.yaml [hunk 1]
```diff
@@ -22,8 +22,6 @@ data:
   Nvidia_mlx5_ConnectX-5_Ex: 15b3 1019 101a
   Nvidia_mlx5_ConnectX-6: 15b3 101b 101c
   Nvidia_mlx5_ConnectX-6_Dx: 15b3 101d 101e
-  Nvidia_mlx5_ConnectX-6_Lx: 15b3 101f 101e
```
Comment (zshi-redhat) [inline manifests/4.10/supported-nic-ids_v1_configmap.yaml:25]: Why removing these two?

Comment (pliurh) [reply manifests/4.10/supported-nic-ids_v1_configmap.yaml:25]: It was synced from upstream. I think we shall keep it unchanged.

Comment (pliurh) [reply manifests/4.10/supported-nic-ids_v1_configmap.yaml]: fixed

**Comment by zshi-redhat** [comment]
we would also need this change in file `deployment/sriov-network-operator/crds/sriovnetwork.openshift.io_sriovnetworknodestates.yaml`?

**Comment by pliurh** [comment]
> we would also need this change in file `deployment/sriov-network-operator/crds/sriovnetwork.openshift.io_sriovnetworknodestates.yaml`?

Yes, but not in this PR. That file is only used by the upstream installation.

**Comment by zshi-redhat** [comment]
> > we would also need this change in file `deployment/sriov-network-operator/crds/sriovnetwork.openshift.io_sriovnetworknodestates.yaml`?
> 
> Yes, but not in this PR. That file is only used by the upstream installation.

Do we already have an upstream PR for this change?

**Pull request merged**